### DISCUSSION TITLE
Azure Monitor: add support for Microsoft.EventGrid/systemTopics namespace

### DISCUSSION THREAD

**Discussion by KristapsT** [grafana/grafana#40695] [Data Sources]
Azure Monitor data source currently does not support Microsoft.EventGrid/systemTopics namespace.

As far as I know, the metrics from this namespace are the same that come from Microsoft.EventGrid/topics which is already supported.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
